### ISSUE TITLE
Option to load GUI defaults when customizing qemu options 

### ISSUE THREAD

**Issue by tiiiecherle** [utmapp/UTM#2308]
When activating 
`Advanced: Bypass configuration and manually specify arguments`
all options set by GUI get deleted and it starts with an almost empty set.

It would be nice to have a button "load GUI values" when the option is active to start with all options set in the GUI but editable to customization. This would make it a lot easier to slightly modify the GUI values without specifying all options manually.

Thanks

**Comment by osy** [comment]
That’s exactly the intended purpose of advanced option. It’s a bypass for the GUI for advanced users. If you need anything from the GUI you shouldn’t be checking that box.

**Comment by tiiiecherle** [comment]
Perhaps I just did not explain correctly. I got the intention of the setting.
But activating `Advanced: Bypass configuration and manually specify arguments` leads to all fields getting cleared.
In my opinion it would be easier if it would just leave all fields filled out with the gui values but makes them editable.
Otherwise I would have to configure a lot of lines for only changing one parameter. I think this would help other users, too.
